targets.json

```diff
@@ -794,6 +794,15 @@
                 "firmware": "Unified_ESP32_900_TX",
                 "prior_target_name": "Unified_ESP32_900_TX"
             }
```

Comment:
You need a comma on the end of this line

Reply:
Good catch, fixed.